### ISSUE TITLE
Stripping the HTML file

### ISSUE THREAD

**Issue by Sfshaza** [dart-lang/dart-pad#647]
When DartPad displays the HTML file, it results in some unattractive indentation due to the way it's stripping the header.

So, this file:

```
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Pirate badge</title>
    <script async type="application/dart" src="main.dart"></script>
    <script async src="packages/browser/dart.js"></script>
    <link rel="stylesheet" href="styles.css">
  </head>

  <body>
    <h1>Pirate badge</h1>

    <div class="widgets">
      TO DO: Put the UI widgets here.
    </div>
    <div class="badge">
      <div class="greeting">
        Arrr! Me name is
      </div>
      <div class="name">
        <span id="badgeName"> </span>
      </div>
    </div>
  </body>
</html>
```

appears like this in DartPad:

```
<h1>Pirate badge</h1>

    <div class="widgets">
      TO DO: Put the UI widgets here.
    </div>
    <div class="badge">
      <div class="greeting">
        Arrr! Me name is
      </div>
      <div class="name">
        <span id="badgeName"> </span>
      </div>
    </div>
```

The first problem, is that the file is stripped up to the `<h1>` tag which changes the indentation of that line. This looks odd.

But let's say that you fixed the problem, so that it looked like this:

```
    <h1>Pirate badge</h1>

    <div class="widgets">
      TO DO: Put the UI widgets here.
    </div>
    <div class="badge">
      <div class="greeting">
        Arrr! Me name is
      </div>
      <div class="name">
        <span id="badgeName"> </span>
      </div>
    </div>
```

That also looks odd, as the entire section is now indented.

So, it would be nice if:
1. You didn't strip the leading characters only on the first line.
2. If you adjusted the indentation of the section to shift the entire segment to the left.

In other words, we'd like to see this in DartPad:

```
<h1>Pirate badge</h1>

<div class="widgets">
  TO DO: Put the UI widgets here.
 </div>
 <div class="badge">
  <div class="greeting">
    Arrr! Me name is
  </div>
  <div class="name">
    <span id="badgeName"> </span>
  </div>
</div>
```

@kwalrath

**Comment by RedBrogdon** [comment]
I'm not seeing any odd indentation in the Sunflower sample. @Sfshaza, can you check to see if this is still an issue?

**Comment by parlough** [comment]
As outlined in https://github.com/dart-lang/dart-pad/issues/2808, support for including a separate HTML has been dropped, so I'm going to close this. Thanks!
